app/stac_api/views.py

```diff
@@ -301,16 +317,19 @@ def get(self, request, *args, **kwargs):
     # Here the etag is only added to support pre-conditional If-Match and If-Not-Match
     @etag(get_item_etag)
     def put(self, request, *args, **kwargs):
+        debug_print_request_data(self)
         return self.update(request, *args, **kwargs)
 
     # Here the etag is only added to support pre-conditional If-Match and If-Not-Match
     @etag(get_item_etag)
     def patch(self, request, *args, **kwargs):
+        debug_print_request_data(self)
         return self.partial_update(request, *args, **kwargs)
 
     # Here the etag is only added to support pre-conditional If-Match and If-Not-Match
     @etag(get_item_etag)
     def delete(self, request, *args, **kwargs):
+        debug_print_request_data(self)
```

Comment:
In delete we don't expect any body and don't do anything about it so no need to log it.